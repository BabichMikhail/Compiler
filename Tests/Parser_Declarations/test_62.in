function a(const a : integer) : integer ;
begin 
end;