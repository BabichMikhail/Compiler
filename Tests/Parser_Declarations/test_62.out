Function   a   2
   Var   a
      Type   integer

   Var   Result
      Type   integer

   begin
   end

